proj/mnv2_first/renode/hps.resc

```diff
@@ -6,6 +6,6 @@ machine StartGdbServer 10001
 showAnalyzer sysbus.uart
 showAnalyzer sysbus.uart Antmicro.Renode.Analyzers.LoggingUartAnalyzer
 
-sysbus LoadELF $ORIGIN/software.elf
+sysbus LoadELF $ORIGIN/../build/software.elf
```

Comment:
@tcal-x this is something we need to think about. HPS repl is not well generated by litex-renode. It stays here, but it affects the flow of the CI

Reply:
I think it's an issue on the litex-renode side though, we'll try to learn more